internal/cypress/docker/runner.go

```diff
@@ -3,24 +3,30 @@ package docker
 import (
 	"context"
 	"encoding/json"
+	"errors"
 	"fmt"
-	"github.com/rs/zerolog/log"
-	"github.com/saucelabs/saucectl/cli/command"
-	"github.com/saucelabs/saucectl/cli/progress"
-	"github.com/saucelabs/saucectl/cli/runner"
-	"github.com/saucelabs/saucectl/cli/streams"
-	"github.com/saucelabs/saucectl/internal/cypress"
 	"io"
 	"io/ioutil"
 	"os"
 	"path"
 	"path/filepath"
 	"strings"
+
+	"github.com/rs/zerolog/log"
+
+	"github.com/saucelabs/saucectl/cli/command"
+	"github.com/saucelabs/saucectl/cli/progress"
+	"github.com/saucelabs/saucectl/cli/runner"
+	"github.com/saucelabs/saucectl/cli/streams"
+	"github.com/saucelabs/saucectl/internal/cypress"
 )
 
 // SauceRunnerConfigFile represents the filename for the sauce runner configuration.
 const SauceRunnerConfigFile = "sauce-runner.json"
 
+// ErrEmptyImageProperty represents error regarding empty image docker image name property in the config file.
+var ErrEmptyImageProperty = errors.New("Docker image name property was not specified in your config file")
```

Comment:
Canonical go errors start with a lower case.
```suggestion
var ErrEmptyImageProperty = errors.New("docker image name property was not specified in your config file")
```

Reply:
I removed this error and left a common error message regarding the empty docker image.